__tests__/api/app.test.js

```diff
@@ -66,4 +65,54 @@ describe('Testing app', () => {
       });
     });
   });
+
+  describe('Endpoint /api/articles/:article_id', () => {
+    describe('Method GET', () => {
+      test('GET: 200 status', () => {
+        return request(app).get('/api/articles/1').expect(200);
+      });
+
+      test('GET: 200 status responds with an article object data', () => {
+        return request(app)
+          .get('/api/articles/1')
+          .then(({ body }) => {
+            const article = body;
+
+            const expectArticle = {
+              article: {
```

Comment:
slight issue with deconstruction here,  if you change line 79 to be `const { article } = body;` then you won't need to have the key of _article_ on line 82

Reply:
Done!